lib/ansible/modules/replace.py

```diff
@@ -175,6 +182,15 @@
     path: /etc/ssh/sshd_config
     regexp: '^(?P<dctv>ListenAddress[ ]+)(?P<host>[^\n]+)$'
     replace: '#\g<dctv>\g<host>\n\g<dctv>0.0.0.0'
+
+- name: Replacing first 2 matches
+  ansible.builtin.replace:
+    path: "/etc/httpd/conf.d/auth.conf"
+    after: "<Location /2>"
+    regexp: 'Require\s+[^\n]+$'
+    replace: 'Require ip {{ ips | join(" ") }}'
+    numberofmatches: 1
```

Comment:
Consider using `count` to reflect the actual keyword parameter that you are using.